libraries/node-core-library/src/test/Path.test.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+/// <reference types='mocha' />
+
+import { Path } from '../Path';
+import { assert } from 'chai';
+
+describe('Path', () => {
+  describe('Test', () => {
+    it('Windows paths', () => {
+      assert.isTrue(Path.isUnder('C:\\a\\b.txt', 'C:\\a'), '1');
+      assert.isTrue(Path.isUnder('C:\\a\\b.txt', 'C:\\a\\'), '2');
+      assert.isTrue(Path.isUnder('C:\\a\\b\\c.txt', 'C:\\a'), '3');
+
+      assert.isFalse(Path.isUnder('C:\\a\\b.txt', 'C:\\b'), '4');
+      assert.isFalse(Path.isUnder('C:\\a\\b.txt', 'C:\\b\\'), '5');
+      assert.isFalse(Path.isUnder('C:\\a\\b\\c.txt', 'C:\\b'), '6');
+    });
+    it('Unix paths', () => {
+      assert.isTrue(Path.isUnder('/a/b.txt', '/a'), '1');
+      assert.isTrue(Path.isUnder('/a/b.txt', '/a/'), '2');
+      assert.isTrue(Path.isUnder('/a/b/c.txt', '/a'), '3');
+
+      assert.isFalse(Path.isUnder('/a/b.txt', '/b'), '4');
```

Comment:
>'/a/b.txt' [](start = 34, length = 10)

Add tests for casing

Reply:
Won't fix -- that will NOT work on Linux, and isn't really testing a meaningful aspect of the implementation (given that we assume the NodeJS API does the right thing).